README.md

```diff
@@ -1,76 +1,71 @@
 
-## Important documents and contacts
+Below are the list of sub-projects:
 
-This repository contains all the project files for the organization, and the following project files are important for understanding the OpenEBS project, and how we operate:
+- [Local PV Hostpath](https://github.com/openebs/dynamic-localpv-provisioner/)
+- [Local PV ZFS](https://github.com/openebs/zfs-localpv)
+- [Local PV LVM](https://github.com/openebs/lvm-localpv)
+- [Mayastor](https://github.com/openebs/mayastor)
 
-* [OpenEBS Vision](/VISION.md) Defines the vision, mission statement; what is in-scope and out-of-scope for the project
-* [Governance](/GOVERNANCE.md) How the project operates
-* [Contributing](/CONTRIBUTING.md) How to contribute, guidelines for being a contributor
-<BR>
+## Who we are
 
-## Project Leadership team
+We are a community of innovative Data Storage engineers that contribute to this project. OpenEBS is the result of hundreds of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. Anyone who is passionate about storage and data is welcome here. <BR>
 
-This Community is managed by the OpenEBS ```Maintainers``` and <kbd>**Senior leaders**</kbd> within the OpenEBS project team. We liaise with the Linux Foundation and CNCF on project, governance
-and operational matters. We curate the daily operations of the project, product, roadmaps, initiatives, all engineering/code activities and all events (including conferences).
-<BR>
+## Community
 
-## Who we are
+This Community repository is the ```central location``` for project info and files like our governance, contribution, code of conduct, security and vision for the OpenEBS organization laid out by the Maintainers. You'll also find a [Top level Issues List](https://github.com/openebs/openebs/issues) for Organization related issues that span across products/projects. We also provide a [main Discussions forum](https://github.com/openebs/openebs/discussions) where **```everyone```** can collaborate and discuss anything.
 
-We are a community of innovative Data Storage engineers that contribute to this project. We are grateful for the major sponsorship provided by [DataCore (also known as DataCore Software)](https://datacore.com), who donate a team of dedicated Dev/QA/Engineers, Product Mgmt, Marketing, Business and Operational leadership as well as significant engineering infrastructure resources and financial funding. Without DataCore OpenEBS would not be the successful product that it is today. OpenEBS is the result of hundreds of amazing individuals, contributors and storage engineers who provide brainstorming ideas, feedback, code reviews and high-quality code to the project. Anyone who is passionate about storage and data is welcome here. <BR>
+- Homepage: [openebs.io](https://openebs.io/)
+- Mailing list: openebs-team@googlegroups.com
```

Comment:
Mailing list?

Reply:
What's the issue?

Reply:
A mailing list is like a address book/email group where you register and then the folks who own the group send you emails monthly. E.g.: https://lkml.org/

Ours is a `Maintainers' email`.